processor/src/main/java/com/linecorp/decaton/processor/LoggingContext.java

```diff
@@ -38,22 +38,29 @@
     public static final String SUBSCRIPTION_ID_KEY = "dt_subscription_id";
     public static final String TASK_KEY = "dt_task_key";
 
-    public LoggingContext(String subscriptionId, TaskRequest request, TaskMetadata metadata) {
-        MDC.put(METADATA_KEY, metadata.toString());
-        MDC.put(TASK_KEY, String.valueOf(request.key()));
-        MDC.put(SUBSCRIPTION_ID_KEY, subscriptionId);
-        MDC.put(OFFSET_KEY, String.valueOf(request.recordOffset()));
-        MDC.put(TOPIC_KEY, request.topicPartition().topic());
-        MDC.put(PARTITION_KEY, String.valueOf(request.topicPartition().partition()));
+    private final boolean enabled;
+
+    public LoggingContext(boolean enabled, String subscriptionId, TaskRequest request, TaskMetadata metadata) {
```

Comment:
I think it's better to keep current constructor to avoid unnecessary breaking change since this class is considered as public API.

Reply:
Good point. Let me follow that.